ibm/mas_devops/roles/cp4d_wds/tasks/embeddedwds.yml

```diff
@@ -0,0 +1,310 @@
+---
+# 1. Check if CP4D is installed and fail if not there
+# -----------------------------------------------------------------------------
+- name: "cp4d/wds: Check if cp4d instance is there"
+  community.kubernetes.k8s_info:
+    api_version: cpd.ibm.com/v1
+    kind: Ibmcpd
+  register: cp4dalllist
+
+- name: "Fail if no cp4d instance is provided before discovery install"
+  when: cp4dalllist.resources| length == 0
```

Comment:
Change this to "assert...that" rather than "when..fail"

Reply:
changed it.